.github/scripts/visualize_stopping_patterns.py

```
@@ -0,0 +1,441 @@
+            f"> **注意:** {len(affected_groups)} 件中 {MAX_GROUPS} 件のみ表示しています。"
+        )
+        parts.append("")
+
+    truncation_msg = "\n\n---\n> **注意:** コメントが長すぎるため、一部が省略されました。"
+    current_length = len("\n".join(parts))
+    truncated = False
+    for section in group_sections:
+        section_length = len("\n") + len(section)
+        if current_length + section_length + len(truncation_msg) > MAX_COMMENT_LENGTH:
+            parts.append(truncation_msg)
+            truncated = True
```

Ruff (0.15.1)

[error] Local variable `truncated` is assigned to but never used

Remove assignment to unused variable `truncated`

(F841)